src/inject.js

```diff
@@ -15,10 +15,9 @@ const inject = () => {
             e.preventDefault()
             e.stopImmediatePropagation()
 
-            // @ts-ignore
-            const activeForm = deviceInterface.getActiveForm()
-            if (activeForm) {
-                activeForm.tooltip.dispatchClick()
+            const activeTooltip = deviceInterface.getActiveTooltip()
+            if (activeTooltip) {
+                activeTooltip.dispatchClick()
             }
```

Comment:
Worth simplifying to just `activeTooltip?.dispatchClick()`? Don't feel strongly, having the `if` could feel more explicit.